### PR TITLE
feat(frontend): update pagination design tm-400

### PR DESCRIPTION
**Pull request by KrasKa2**
Screenshot 2024-03-10 at 14 12 54

### CODE REVIEW

**Comment by GEOFARL** [comment]
I'm not sure, but it seems like chevron icons do not match the design

File: apps/frontend/src/libs/components/pagination/pagination.tsx [hunk 1]
```diff
@@ -41,12 +41,12 @@ const Pagination: React.FC<Properties> = ({
 			<ul className={styles["content"]}>
 				<PaginationItem
 					isDisabled={isFirstPage}
-					label="First"
+					label="<<"
```
Comment (GEOFARL) [inline apps/frontend/src/libs/components/pagination/pagination.tsx:44]: Probably, you need to create two new icons for this arrows

Comment (what1s1ove) [reply apps/frontend/src/libs/components/pagination/pagination.tsx:44]: the label should be user-friendly. use icons instead

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Review by YourTeaChan — APPROVED** [review]

**Review by Vfivew — APPROVED** [review]

File: apps/frontend/src/libs/components/pagination/libs/components/pagination-item/pagination-item.tsx [hunk 2]
```diff
@@ -20,14 +24,16 @@ const PaginationItem: React.FC<Properties> = ({
 		[styles["disabled"] as string]: isDisabled,
 	});
 
+	const content = iconName ? <Icon name={iconName} /> : label;
```
Comment (fshabanov) [inline apps/frontend/src/libs/components/pagination/libs/components/pagination-item/pagination-item.tsx:27]: Always use label, add `hasVisuallyHiddenLabel` prop

**Review by fshabanov — CHANGES_REQUESTED** [review]

**Review by fshabanov — APPROVED** [review]

**Review by ShadowOfTheSpace — APPROVED** [review]

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**